user-dashboard.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800;900&display=swap');
+
+body {
+    background-color: rgba(169, 175, 197, 0.05);
+    font-family: Nunito;
+}
+
+.bg-blue {
+    background-color: #1F59BB;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#1F59BB" to be "#1f59bb" (color-hex-case)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5690830)